llvm/lib/Target/RISCV/MCA/RISCVCustomBehaviour.cpp

```diff
@@ -20,27 +20,14 @@
 
 #define DEBUG_TYPE "llvm-mca-riscv-custombehaviour"
 
-// This brings in a table with primary key of
-// base instruction opcode and lmul and maps
-// to the opcode of the pseudo instruction.
-namespace RISCVVInversePseudosTable {
-using namespace llvm;
-using namespace llvm::RISCV;
-
-struct PseudoInfo {
-  uint16_t Pseudo;
-  uint16_t BaseInstr;
-  uint8_t VLMul;
-  uint8_t SEW;
-};
+namespace llvm {
 
+namespace RISCVVInversePseudosTable {
+using namespace RISCV;
 #define GET_RISCVVInversePseudosTable_IMPL
-#define GET_RISCVVInversePseudosTable_DECL
 #include "RISCVGenSearchableTables.inc"
```

Comment:
We don't need to include this any more, we can use the one in `RISCVMCTargetDesc.h`.

Reply:
Are you sure? It looks like we need to have the include after the `_IMPL` define. The include in `RISCVMCTargetDesc.h` occurs before this and `GET_RISCVVInversePseudosTable_IMPL` is not set yet. RISCVInstrInfo.cpp does it like we have here for `RISCVVPseudosTable_IMPL` for example.

Reply:
I meant we can put `GET_RISCVVInversePseudosTable_DECL` in `RISCVMCTargetDesc.h` and put `GET_RISCVVInversePseudosTable_IMPL` in `RISCVMCTargetDesc.cpp`.